src/rpdk/core/contract/resource_client.py

```diff
@@ -234,6 +235,13 @@ def make_request(desired_resource_state, previous_resource_state, **kwargs):
     def generate_token():
         return str(uuid4())
 
+    @staticmethod
+    def assert_time(start_time, end_time, action):
+        timeout_in_seconds = 60
+        if action in (Action.READ, Action.LIST):
```

Comment:
i'm not sure how useful this assertion will be since the sam cli local cold start time is a lot longer than on the lambda service and varies depending on if the latest docker lambda image is downloaded. we already account for this in the [client](https://github.com/aws-cloudformation/cloudformation-cli/blob/master/src/rpdk/core/contract/resource_client.py#L87). i could see this becoming frustrating for people trying to test if their machine is slow. maybe we can have this as an option to enable?

Reply:
it should not be optional. is there any better way to account for the cold start time when checking the time?  How long is the cold start time?